### PR TITLE
Style:[UIUX-497]disabled filter chips

### PR DESCRIPTION
**Pull request by sidharthjain-zemoso**
# Description
Changed styles of disabled filters to differentiate it from enabled filter.

# Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Could you provide instructions so we can reproduce them? Please also
list any relevant details for your test configuration

Before:
image

After:
image

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The commit message/PR follows our guidelines

# **Other information**:

### CODE REVIEW

**Review by mkinchin-paladin — APPROVED** [review]

**Review by mkinchin-paladin — CHANGES_REQUESTED** [review]
Is there ticket for these changes?

**Comment by sidharthjain-zemoso** [comment]
> Is there ticket for these changes?

Yes we have a ticket - UIUX-497.

**Review by mkinchin-paladin — APPROVED** [review]

**Review by mkinchin-paladin — CHANGES_REQUESTED** [review]
Do not merge, this needs to be discussed with @plyubich

**Review by plyubich — CHANGES_REQUESTED** [review]

File: webapp/src/app/shared/table-filters/table-filters.component.html [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-        [appliedFiltersDict]="appliedFiltersDict[appliedFilter.keyDisplayValue]"
-        [options]="categoryOptions[appliedFilter.keyDisplayValue]"
-        [filtersToExcludeFromCasing]="filtersToExcludeFromCasing"
-        (clear)="clearFilter($event)"
-        (update)="updateFilter($event)"
-        (click)="openFilterCategory(appliedFilter.keyDisplayValue)"
-        (filterSearchTextChange)="handleSearchTextChangeForCategory($event)"
-    ></app-table-filter-chip>
+    >
+        <app-table-filter-chip
```
[the remark this answers is not in the document]
Comment (sidharthjain-zemoso) [reply webapp/src/app/shared/table-filters/table-filters.component.html]: Formatted that block.

**Review by plyubich — APPROVED** [review]

**Pull request merged**